### PR TITLE
feat: add Ticket 2019 info

### PR DESCRIPTION
**Pull request by Farson**
Ticket sale will start on Monday 21.1. Here i added a link to Tito and  the Tito Embeddable JavaScript Widget

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto+Mono:400|Roboto:400,700" rel="stylesheet">
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
     <link rel="icon" href="/assets/images/favicon.png" type="image/png">
+    <script src='https://js.tito.io/v1' async></script>
```
Comment (emrox) [inline _layouts/default.html:14]: can it be loaded within body to not block the site while loading?

File: index.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ title: "Ruby Unconf Hamburg 2019"
 
 <div class="content-section content-section--whitebg" markdown="1">
 
+## Tickets
```
Comment (emrox) [inline index.md]: maybe below the 2019 section because the users should read the basic facts first?

File: index.md [hunk 3]
```diff
@@ -5,6 +5,17 @@ title: "Ruby Unconf Hamburg 2019"
...
+
+### with all tickets you gain access to both conference days as well as free food, drinks and entry to the party.
+
+[Tickets](https://ti.to/ruby-unconf/2019) sale will start on Monday 21 January
```
Comment (emrox) [inline index.md]: shouldn't it be `Ticket sale`? (without plural `s`)

File: _layouts/default.html (hunk 1 above)
Comment (Farson) [reply _layouts/default.html:14]: Isn't it loaded asynchron anyway?

**Pull request merged**